sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -586,6 +598,11 @@ class CustomFormModel(object):
         List of any training errors.
     :ivar list[~azure.ai.formrecognizer.TrainingDocumentInfo] training_documents:
          Metadata about each of the documents used to train the model.
+    :ivar str display_name: Optional user defined model name (max length: 1024).
```

Comment:
Are this docstring the user can see? I would recommend to removing the size restriction because the service could change this at any point and forget to update the swagger. so better let them manage that.